timemachine/fe/mcgregor.py

```diff
@@ -69,20 +69,15 @@ def refine_marcs(g1, g2, new_v1, new_v2, marcs):
 
     if new_v2 == UNMAPPED:
         # zero out rows corresponding to the edges of new_v1
-        for e1 in g1.get_edges(new_v1):
-            # this is equivalent to new_marcs[e1] = np.zeros(marcs.shape[1])
-            new_marcs[e1] = 0
+        new_marcs[g1.get_edges_as_vector(new_v1)] = False
     else:
-        # mask out every row in marcs
```

Comment:
retain line comment?

(the np.where update on vectors of bitmasks a few lines later was a little tricky for me to skim, maybe renaming to avoid overwriting the variable `mask` might be useful. `adj1 = g1.get_edges_as_vector(new_v1); adj2 = g2.get_edges_as_vector(new_v2); mask = np.where(adj1[:, np.newaxis], adj2, ~adj2)` or similar...)

Reply:
Thanks for the suggestion, this looks a lot clearer. Applied in https://github.com/proteneer/timemachine/pull/1233/commits/8178dd2bf6e8e754fe5a373e3e70597872d4a9fd